pydrive2/drive.py

```diff
@@ -47,3 +47,29 @@ def GetAbout(self):
         :returns: A dictionary of Google Drive information like user, usage, quota etc.
         """
         return self.auth.service.about().get().execute(http=self.http)
+        
+
+    @LoadAuth
+    def CopyFile(self, source_file_id, target_folder_id, new_name):
+        """Copy a file to a new location.
+
+        :param source_file_id: ID of the file to copy.
+        :type source_file_id: str.
+        :param target_folder_id: ID of the folder to copy to.
+        :type target_folder_id: str.
+        :param new_name: Name of the new file.
+        :type new_name: str.
+        """
+
+        body = {
+            "parents": [{"kind": "drive#fileLink",
```

Comment:
Q: `drive#fileLink` why do we need specifying `kind` here?

Reply:
you are right pointing it out, because we don't. It works with just the id.